examples/torch/mtsac_MT10.py

```diff
@@ -0,0 +1,100 @@
+"""MTSAC implementation based on Metaworld. Benchmarked on MT10.
+
+https://arxiv.org/pdf/1910.10897.pdf
+"""
+import click
+from metaworld.benchmarks import MT10
+import numpy as np
+from torch import nn
+from torch.nn import functional as F
+
+from garage import wrap_experiment
+from garage.envs import GarageEnv, MultiEnvWrapper, normalize
+from garage.experiment import deterministic, LocalRunner
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.sampler import LocalSampler
+from garage.torch.algos import MTSAC
+from garage.torch.policies import TanhGaussianMLPPolicy
+from garage.torch.q_functions import ContinuousMLPQFunction
+import garage.torch.utils as tu
+
+
+@click.command()
+@click.option('--seed', 'seed', type=int, default=1)
+@click.option('--use_gpu', 'use_gpu', type=bool, default=False)
+@click.option('--gpu', '_gpu', type=int, default=0)
```

Comment:
Why not default `--gpu` to `None` and then only assign it to a gpu if it is non-None?